elitzur-core/src/main/scala/com/spotify/elitzur/validators/ValidationStatus.scala

```diff
@@ -170,3 +173,9 @@ final case class Invalid[+A](x: A) extends PostValidation[A] {
   // Override this so flatten and flatMap skip Invalid records
   override def iterator: Iterator[A] = Iterator()
 }
+
+object Invalid {
+  def of[A](x: PostValidation[A]): Option[Invalid[A]] = Option(x)
```

Comment:
seems overkill but we need a clean way to map from PostValidation to invalid to retrieve invalid fields ( or w.e. we use to provide extra info on invalid records)